### ISSUE TITLE
LabyMod capes

### ISSUE THREAD

**Issue by Madis0** [PepperCode1/AnyCapes#1]
Could you add [LabyMod](https://www.labymod.net) capes to this mod? I don't know the API URL, but you could possibly find it from a [similar Fabric mod](https://github.com/CaelTheColher/Capes).

**Comment by PepperCode1** [comment]
I will make a commit and release a new version in the next few days that will add three more APIs to the default configuration, including LabyMod.

**Comment by PepperCode1** [comment]
Sorry for the lack of updates. I was planning to add more APIs to the default configuration as soon as I released the first version, but I came across a bug that I couldn't figure out. After about a week I gave up trying to solve it and a new version was never released.
The way that LabyMod's cape API works is that even if a player doesn't have a cape, it doesn't return a 404 code. Instead, it returns an invalid image, which messed with Minecraft's internals somehow and ends up displaying a really messed up cape. There was an issue with Wynntils capes as well, but I don't quite remember what it was.
